setup/src/main/java/io/jenkins/jenkinsfile/runner/util/JoinClassLoader.java

```diff
@@ -0,0 +1,90 @@
+package io.jenkins.jenkinsfile.runner.util;// Copyright 2010 Christian d'Heureuse, Inventec Informatik AG, Zurich, Switzerland
+// www.source-code.biz, www.inventec.ch/chdh
+//
+// This module is multi-licensed and may be used under the terms
+// of any of the following licenses:
+//
+//  LGPL, GNU Lesser General Public License, V2 or later, http://www.gnu.org/licenses/lgpl.html
```

Comment:
Note that this code is not compatible w.r.t the license. I put it here only for the experiment (which does not work so far), and it should not be merged as is